src/main/scala/de/heikoseeberger/sbtheader/HeaderPlugin.scala

```diff
@@ -54,6 +54,7 @@ object CommentStyleMapping {
 
 object HeaderKey {
   val headers = settingKey[Map[String, (Regex, String)]]("Header pattern and text by extension; empty by default")
+  val excludes = settingKey[Seq[String]]("File patterns for files to be excluded; empty by default")
```

Comment:
Are you sure about that? I thought predef included the immutable variant. `Seq` is the preference all over sbt (I prefer `List` but that's just me).